README.md

```diff
@@ -34,11 +53,27 @@ class SomeAPI < Grape::API do
     security required: %w[foo/bar.baz foo/bar.qux]
   end
   params do
-    optional :normal
-    optional :nested, type: Hash do
-      optional :sub
+    optional :filter, type: Array do
+      optional :foo_id, type: [Integer]
     end
   end
   get('/foos') { [Mock::Foo.new(foo_id: 1)] }
+
+  desc 'Create Bar' do
+    success Mock::Bar::Entity
+  end
+  params do
+    optional :foos, type: Array[JSON] do
+      optional :foo_id, type: Integer
+    end
+  end
+  post('/bars/:id') do
+    Mock::Bar.new(
+      bar_id: 2,
+      foos: [Mock::Foo.new(foo_id: 1)],
+    )
+  end
 end
 ```
+
+Example of produced apidoc for this app: [spec/fixtures/golden.md](spec/fixtures/golden.md)
```

Comment:
Can we have the mock app (entities, api) somewhere there as well? (linkable, don't think it's OK to place ruby classes in fixtures dir)

Reply:
@dim any suggestions? ^^

Reply:
you can do what I did in grape-app and have a `spec/scenario`?

Reply:
I'll try, but in another PR.

Reply:
https://github.com/bsm/grape-apidoc/pull/11